### DISCUSSION TITLE
How do I control colors of the menus and side-bar?

### DISCUSSION THREAD

**Discussion by xero-lib** [neoclide/coc.nvim#4786] [Q&A]
I'm extremely new to CoC and have no idea how to change colors in vim period even without CoC.

I can barely read the default colors, and was wondering how I might be able to change them?

![image](https://github.com/neoclide/coc.nvim/assets/54485853/fd229bcf-355c-42d6-b641-7b98c128303c)

**Comment by fannheyward** [comment]
Check `:h CocPum` fot the highlight group used.

**Comment by xero-lib** [comment]
> Check `:h CocPum` fot the highlight group used.

Is there any documentation or wiki for the different groups and configuration formats?
